### PR TITLE
CDP-363 Update CDP Dashboard to show under /borrow route

### PR DESCRIPTION
**Pull request by sirromdev**

### CODE REVIEW

File: src/pages/Auth.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            {lang.landing_page.wallet_connect}
+          </IconButton>
+          {/* <ReadOnlyConnect /> */}
+        </Grid>
+      </Flex>
+    </PageContentLayout>
+  );
+}
+
+export default hot(Auth);
```
Comment (sirromdev) [inline src/pages/Auth.js]: Moved wallet component to own file. This probably covers some of the work for CDP-361

File: src/utils/constants.js [hunk 2]
```diff
@@ -18,3 +18,8 @@ export const AccountTypes = {
   TREZOR: 'trezor',
   METAMASK: 'browser'
 };
+
+export const Routes = {
+  MCD: 'borrow',
+  DSR: 'save'
+};
```
Comment (sirromdev) [inline src/utils/constants.js]: Not sure if this is the best place for these and that the names for the keys are ok?

Comment (levity) [reply src/utils/constants.js]: The key names might as well be `BORROW` and `SAVE` too

**Comment by sirromdev** [comment]
Seems to be an issue with name aliasing that is causing the build to fail. 

Reason for not building

**Comment by levity** [comment]
> Seems to be an issue with name aliasing that is causing the build to fail.
> 
> Reason for not building

hmm, i wonder if it's the capital letters or the length. i just pushed a commit to develop that lowercases the branch name; merge that in and see if it fixes it.

**Pull request merged**